web/ui/src/features/widgets/Table.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import styles from './Table.module.css';
+
+interface Props {
+  tableHeaders: string[];
+  style?: React.CSSProperties;
+  renderTableData: () => JSX.Element[];
+}
+
+/**
+ * Simple table component that accept a custom header, and custom render
+ * function for the table data
+ */
+const Table = ({ tableHeaders, style = {}, renderTableData }: Props) => {
```

Comment:
Do we not have a `Table` in `@grafana/ui`?

Reply:
We do, but I find the Table from grafana ui more than necessary for a custom table, and harder to use in data conversion. I might using it in the future if there is more complex table needed in the agent app, but for now this simple Table widget will do.